### PR TITLE
GCU test for BGP prefix

### PR DESCRIPTION
**Pull request by wen587**
### Description of PR

[Done]Wait for yang model https://github.com/Azure/sonic-buildimage/pull/10142
Wait for issue fix: https://github.com/Azure/sonic-buildimage/issues/10141
Summary: Testcase of bgp allowed prefix config for generic updater apply-patch
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
End to End test support for Generic Updater apply-patch
This PR is to verify the usage of 'config apply-patch' works on bgp allowed prefix config

#### How did you do it?
Add bgp prefix config to dut and check if config change as expected

#### How did you verify/test it?
Run test of sonic-mgmt/tests/generic_config_updater/test_bgp_prefix.py on KVM

### CODE REVIEW

File: tests/generic_config_updater/test_bgp_prefix.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+import pytest
+import re
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_failure, expect_op_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+pytestmark = [
+    pytest.mark.topology('t0'),  #TODO: t0?
```
Comment (ghooo) [inline tests/generic_config_updater/test_bgp_prefix.py:11]: I think T0s should be enough

**Review by ghooo — APPROVED** [review]

**Review by ghooo — CHANGES_REQUESTED** [review]
check open comment

File: tests/generic_config_updater/test_bgp_prefix.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
```
Comment (qiluo-msft) [inline tests/generic_config_updater/test_bgp_prefix.py:11]: >#TODO: t0 [](http://example.com/codeflow?start=33&length=9)

The TODO comment does not make sense. You already covered t0.

File: tests/generic_config_updater/test_bgp_prefix.py (hunk 1 above)
Comment (wen587) [reply tests/generic_config_updater/test_bgp_prefix.py]: Fixed.

File: tests/generic_config_updater/test_bgp_prefix.py (hunk 2 above)
Comment (wen587) [reply tests/generic_config_updater/test_bgp_prefix.py]: fixed

**Review by qiluo-msft — APPROVED** [review]
LGTM. Please check with other reviewers.

**Review by ghooo — APPROVED** [review]

**Pull request merged**